app/main.py

```diff
@@ -1,2 +1,112 @@
+from __future__ import annotations
+
+from fractions import Fraction
+from typing import Any, Iterator
+
+INITIAL_CAPACITY = 8
```

Comment:
these should be class variables  inside of Dictionary since it is part of it implementation